frontend/src/metabase/query_builder/components/QueryModals.jsx

```diff
@@ -237,3 +257,20 @@ export default class QueryModals extends React.Component {
     ) : null;
   }
 }
+
+const QuestionMoveToast = ({ isModel, collectionId }) => {
```

Comment:
Copied from 
https://github.com/metabase/metabase/blob/6852378d1e826549723652b5568d7dc7cf30d8f2/frontend/src/metabase/dashboard/components/DashboardMoveModal.jsx#L54-L61

so we show the same style of toast message after moving saved questions, models, and dashboards.
![image](https://user-images.githubusercontent.com/1937582/177760478-4ad22dcc-275d-440c-9115-23efba27cca5.png)

Reply:
Looks good, but I think we should at least copy `ToastRoot` from where we use it and put it here. We could also move `QuestionMoveToast` to a separate file (and folder), so there would be a corresponding `QuestionMoveToast.tsx`.

Reply:
Fixed.